doc/classes/Object.xml

```diff
@@ -185,34 +191,35 @@
 			<argument index="0" name="name" type="String">
 			</argument>
 			<description>
-				Return a metadata from the object.
+				Returns [code]metadata[/code] from the Object.
 			</description>
 		</method>
 		<method name="get_meta_list" qualifiers="const">
 			<return type="PoolStringArray">
 			</return>
 			<description>
-				Return the list of metadata in the object.
+				Returns the list of [code]metadata[/code] for this Object.
 			</description>
 		</method>
 		<method name="get_method_list" qualifiers="const">
 			<return type="Array">
 			</return>
 			<description>
+				Returns an [Array] of dictionaries for the methods and their signatures for this Object.
 			</description>
 		</method>
 		<method name="get_property_list" qualifiers="const">
 			<return type="Array">
 			</return>
 			<description>
-				Return the list of properties as an array of dictionaries, dictionaries contain: name:String, type:int (see TYPE_* enum in [@Global Scope]) and optionally: hint:int (see PROPERTY_HINT_* in [@Global Scope]), hint_string:String, usage:int (see PROPERTY_USAGE_* in [@Global Scope]).
+				Returns the list of properties as an [Array] of dictionaries, dictionaries contain: name:String, type:int (see TYPE_* enum in [@Global Scope]) and optionally: hint:int (see PROPERTY_HINT_* in [@Global Scope]), hint_string:String, usage:int (see PROPERTY_USAGE_* in [@Global Scope]).
```

Comment:
Second phrase should be a separate sentence. Add spaces after colons. I think we should use enum links, but I haven't been able to confirm if they're working correctly yet.